tests/test_scripts.py

```diff
@@ -376,6 +436,35 @@ def test_train_adversarial(tmpdir, named_configs, command):
     _check_train_ex_result(run.result)
 
 
+@pytest.mark.parametrize("command", ("airl", "gail"))
+def test_train_adversarial_warmstart(tmpdir, command):
+    named_configs = ["cartpole"] + ALGO_FAST_CONFIGS["adversarial"]
+    config_updates = {
+        "common": dict(log_root=tmpdir),
+        "demonstrations": dict(rollout_path=CARTPOLE_TEST_ROLLOUT_PATH),
+    }
+    run = train_adversarial.train_adversarial_ex.run(
```

Comment:
I think `train_adversarial` can be warmstarted from the expert in `CARTPOLE_TEST_POLICY_PATH` without needing to train AIRL/GAIL then warmstart from that policy, which might simplify things.

Reply:
If I use the CARTPOLE_TEST_POLICY_PATH, tests don't pass with a horizon of 10, getting `AssertionError: No updates (need at least 512 timesteps, have only total_timesteps=10)!`. Thoughts?

Reply:
Huh, where's the horizon of 10 coming from? CartPole should have 500 horizon by default.